translations/w3c_website_templates_bundle+intl-icu.en.yaml

```diff
@@ -191,3 +191,45 @@ events:
     website: Website
 alumni:
     date_range: "{start} to {end}"
+error:
+    intro: The server returned a "{status_code} {status_text}" error.
```

Comment:
```suggestion
    intro: The server returned a "{status_code} - {status_text}" error.
```

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319

Reply:
Addressed by @deniak in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319